### DISCUSSION TITLE
hugo-add-component

### DISCUSSION THREAD

**Discussion by giscus** [leegical/leegical.github.io#1] [Announcements]
# posts/hugo-add-component/

本文主要记录了如何引入 giscus 为博客添加评论功能。用户登陆Github账号后即可评论文章。

https://haoyep.com/posts/hugo-add-component/

**Comment by wsm6636** [comment]
好耶

**Comment by fqw1727252209** [comment]
现实生活中，我可以说很牛批，但是这里是gay站，我只能说，还得练！
